adapters/rubicon/rubicon_test.go

```diff
@@ -1046,7 +1046,8 @@ func TestOpenRTBRequest(t *testing.T) {
 		},
 	}
 
-	reqs, errs := bidder.MakeRequests(request)
+	reqInfo := adapters.ExtraRequestInfo{}
+	reqs, errs := bidder.MakeRequests(request, reqInfo)
 
```

Comment:
Again, if for some reason, a `RequestType` field can't be added to the `openrtb.BidRequest` struct itself, can we save us the hassle of having an extra local variable here and get rid of the `reqInfo` declaration? If so, line 1049 would be gone and we could just have:
```
1050        reqs,errs := bidder.MakeRequests(request, adapters.ExtraRequestInfo{})
```